Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -20,6 +20,7 @@ import PackageGraph
 /// The parameters required by `PIFBuilder`.
 public struct PIFBuilderParameters {
 
+    // 👀 not sure if this should go also, how is it used?
```

Comment:
👀

Reply:
I don't have any history here at all.  I'll take a look at the PIF generation and see how it works, and comment again.